src/utils/string-width.ts

```diff
@@ -0,0 +1,9 @@
+export const stringWidth = (stringThing, size = 12) => {
```

Comment:
If we do port this to IHME-UI, I think this function is already there. Aaaaand this `if` clause could be a big improvement to what is there! Nice find!